resources/run_adf.sh

```diff
@@ -209,7 +258,7 @@ else
   fi
 fi
 
-check_mongohoused $MONGO_DOWNLOAD_DIR
+check_mongohoused $MONGOSH_DOWNLOAD_DIR
```

Comment:
```suggestion
check_mongohoused
```
Looks like with your change we don't need the download dir passed in.  Try without these and see if it works.